mico-admin/src/app/data-view/mico-data-string/mico-data-string.component.html

```diff
@@ -1,3 +1,12 @@
-<mat-list-item class="mat-list-item">
-    <b>{{config.title}}:</b> &nbsp; {{dataValue}}
-</mat-list-item>
+<!-- Reconsider the boundary -->
+<div *ngIf="dataValue?.length > 100; else elseBlock">
+    <mat-list-item style="height: auto; min-height: 48px">
+        <b>{{config.title}}:</b> &nbsp; {{dataValue}}
```

Comment:
use margin right css attribute instead of nbsp to space title and value. Maybe use a span tag instead of b for title.